webview/src/plots/components/customPlots/CustomPlotsGrid.tsx

```diff
@@ -1,10 +1,8 @@
-import React, { RefObject } from 'react'
```

Comment:
When I tested the demo for the first time, the custom plots didn't work:

https://github.com/iterative/vscode-dvc/assets/43496356/4fce24fd-97f1-47d0-accf-f565e4542c9c

But when I switched to `main` then back to this pr, it was working correctly and I can no longer recreate the plots breaking. 

Not sure if the plots breaking was due to this pr or just something weird happening on my end 🤔

Reply:
I added `useLayoutEffect` after all. I reset my project a couple of times to make sure it does not happen again.